integration-libs/digital-payments/src/checkout/adapters/config/occ-digital-payments-endpoint.model.ts

```diff
@@ -0,0 +1,9 @@
+import { OccEndpoint } from '@spartacus/core';
+
+export interface digitalPaymentsEndpoints {
```

Comment:
Interface name should start with capital letter and for convention I would add Occ in the name
```suggestion
export interface DigitalPaymentsOccEndpoints {
```

Reply:
done